packages/robot/odahuflow/robot/profiler_loader.py

```diff
@@ -52,7 +52,7 @@ def get_variables(profile=None) -> typing.Dict[str, str]:
         variables = {}
 
         try:
-            host_base_domain = "odahu.{}.{}".format(data.get('cluster_name'), data.get('root_domain'))
+            host_base_domain = "odahu.{}".format(data['dns']['domain'])
```

Comment:
We decided that `dns.domain` will contain the prefix.

Reply:
You're right, thanks.